kyo-prelude/shared/src/main/scala/kyo/kernel/ArrowEffect.scala

```diff
@@ -169,33 +169,25 @@ object ArrowEffect:
             I2[_],
             O2[_],
             E2 <: ArrowEffect[I2, O2],
-            I3[_],
-            O3[_],
-            E3 <: ArrowEffect[
-                I3,
-                O3
-            ],
             A,
             S,
             S2
         ](
             inline tag1: Tag[E1],
             inline tag2: Tag[E2],
-            inline tag3: Tag[E3],
-            v: A < (E1 & E2 & E3 & S),
+            v: A < (E1 & E2 & S),
             context: Context
         )(
             inline stop: => Boolean,
-            inline handle1: [C] => (I1[C], Safepoint ?=> O1[C] => A < (E1 & E2 & E3 & S & S2)) => A < (E1 & E2 & E3 & S & S2),
-            inline handle2: [C] => (I2[C], Safepoint ?=> O2[C] => A < (E1 & E2 & E3 & S & S2)) => A < (E1 & E2 & E3 & S & S2),
-            inline handle3: [C] => (I3[C], Safepoint ?=> O3[C] => A < (E1 & E2 & E3 & S & S2)) => A < (E1 & E2 & E3 & S & S2)
-        )(using inline _frame: Frame, inline flat: Flat[A], safepoint: Safepoint): A < (E1 & E2 & E3 & S & S2) =
```

Comment:
This method is used only by `IOTask` and it doesn't need to handle `IO` anymore.